### PR TITLE
fix(input): fixes alignment issues between textareas and inputs

### LINKED ISSUE #5462: md-input-container misalign
1.0.0-rc2

http://codepen.io/0101adm/pen/WQyXmM

### PR DESCRIPTION
**Pull request by robertmesserle**
Closes #5462 

@ThomasBurleson @jelbourn

### CODE REVIEW

File: src/components/input/input.scss [hunk 1]
```diff
@@ -156,6 +157,9 @@ md-input-container {
     width: 100%;
     box-sizing: border-box;
 
+    // Hacky fix to force proper alignment between `input` and `textarea`
```
Comment (jelbourn) [inline src/components/input/input.scss]: Could you expand on this a little more? I.e., "Without this... "

**Comment by ThomasBurleson** [comment]
@robertmesserle - this appears to have broken the Input demos.

**Comment by robertmesserle** [comment]
@ThomasBurleson How so?  And what browser?  (They are working fine for me in Chrome)

**Comment by robertmesserle** [comment]
@ThomasBurleson This should be good to merge now.

**Pull request closed without merging**